### PR TITLE
Add device id and session id getters

### PR DESCRIPTION
**Pull request by budo385**
Hi can you please check out the device id and session id getters. 
Thank you!

### CODE REVIEW

**Comment by yuhao900914** [comment]
Hi @budo385, thanks for making contribution to Amplitude-Flutter SDK. 
I will review your code and update you soon. 
Thanks.

File: ios/Classes/SwiftAmplitudeFlutterPlugin.swift [hunk 1]
```diff
@@ -155,6 +155,14 @@ import Amplitude
                     Amplitude.instance(withName: instanceName).uploadEvents()
                     result(true)
 
+                case "deviceId":
+                    let deviceId = Amplitude.instance(withName: instanceName).deviceId
+                    result(deviceId)
+
+                case "sessionId":
+                    let sessionId = Amplitude.instance(withName: instanceName).getSessionId()
+                   result(sessionId)
```
Comment (haoliu-amp) [inline ios/Classes/SwiftAmplitudeFlutterPlugin.swift:164]: indent. @yuhao900914

File: lib/amplitude.dart [hunk 2]
```diff
@@ -247,6 +247,16 @@ class Amplitude extends _Amplitude {
         'uploadEvents', jsonEncode(_baseProperties()));
   }
 
+  /// deviceId.
```
Comment (haoliu-amp) [inline lib/amplitude.dart:250]: Doc details @yuhao900914

File: lib/amplitude.dart [hunk 3]
```diff
@@ -247,6 +247,16 @@ class Amplitude extends _Amplitude {
...
+  Future<String> deviceId() async {
+    return await _channel.invokeMethod('deviceId', jsonEncode(_baseProperties()));
+  }
+
+  /// sessionId.
```
Comment (haoliu-amp) [inline lib/amplitude.dart:255]: Same @yuhao900914

File: example/ios/Podfile [hunk 4]
```diff
@@ -1,89 +0,0 @@
-# Uncomment this line to define a global platform for your project
-platform :ios, '10.0'
```
Comment (yuhao900914) [inline example/ios/Podfile:2]: The Podfile cannot be deleted. It is a specification that describes the dependencies of the targets of the flutter example project.  Could you please revert it back?

**Comment by yuhao900914** [comment]
Hi @budo385. Thanks for your contribution to Amplitude-Flutter. Want to give you update that we released the v3.1.0 to make sessionId and deviceId available through getSessionId() and getDeviceId().

**Pull request closed without merging**